ando/tests/test_isAnDO.py

```diff
@@ -23,37 +23,37 @@ def setUp(self):
     '''testing dataset001 folder'''
     def test_AnDO_dataset_1(self):
         directory = currpath+"/dataset001/Landing"
-        self.assertEqual(andoE.is_AnDO(directory), False)
+        self.assertEqual(andoE.is_AnDO(directory,False), False)
 
     '''testing dataset002 folder'''
     def test_AnDO_dataset_2(self):
         directory = currpath+"/dataset002/exp-Landing"
-        self.assertEqual(andoE.is_AnDO(directory), False)
+        self.assertEqual(andoE.is_AnDO(directory,False), False)
```

Comment:
idem!!! I stop here... can you check everywhere that you have a space between the different arguments of a function (there are several other ones below...)